templates/liquibase.nuspec.TEMPLATE

```diff
@@ -22,7 +22,7 @@ Liquibase is an open source, database-independent library for tracking, managing
     <projectSourceUrl>https://github.com/liquibase/liquibase</projectSourceUrl>
     <mailingListUrl>http://www.liquibase.org/blog/</mailingListUrl>
     <dependencies>
-      <dependency id="openjdk8"/>
+      <dependency id="openjdk" version="17.0.2"/>
```

Comment:
can we update this dep to temurin 17 since openjdk is deprecated?

Reply:
done @jnewton03